xml/System.Diagnostics/DiagnosticListener.xml

```diff
@@ -279,9 +279,14 @@
         <Parameter Name="isEnabled" Type="System.Func&lt;System.String,System.Object,System.Object,System.Boolean&gt;" Index="1" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="observer">To be added.</param>
-        <param name="isEnabled">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="observer">An <see cref="T:System.IObserver" /> subscriber.</param>
+        <param name="isEnabled">Filters events based on their name (<see cref="T:System.String" />) and up to two context object (which can be <see langword="null" />).  
+A particular instrumentation site has the option of calling one or more <see cref="M:System.Diagnostics.DiagnosticListener.IsEnabled" /> overloads  in which it passes the name of the event and up to two other (instrumentation site specific) objects as arguments. If any of these <see cref="M:System.Diagnostics.DiagnosticListener.IsEnabled" /> calls are made then this <paramref name="isEnabled" /> predicate is invoked with passed values (if shorter overloads are used, <see langword="null" /> is passed for missing context objects).
```

Comment:
These are remarks, not part of the parameter name description.

Reply:
Let me commit your suggestions first and then I will update these lines because they need several substitutions.